src/buttons/docs/readme.md

```diff
@@ -1,3 +1,36 @@
-There are two directives that can make a group of buttons behave like a set of checkboxes, radio buttons, or a hybrid where radio buttons can be unchecked.
+With the buttons directive, we can make a group of buttons to behave like a set of checkboxes (`uib-btn-checkbox`) or to behave like a set of radio buttons (`uib-btn-radio`).
```

Comment:
Didn't highlight because it is not talking about the directive in concrete, but html buttons. 

The other, yeah :)